_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3195,13 +3146,7 @@ public static function service_api_invalid_service_response() {
 	 * @param string $service The service the API key is for.
 	 */
 	public static function validate_service_api_key( $key = null, $service = null ) {
-		$validation = false;
-		switch ( $service ) {
-			case 'mapbox':
-				$validation = self::validate_service_api_key_mapbox( $key );
-				break;
-		}
-		return $validation;
+		return WPCOM_REST_API_V2_Endpoint_Service_API_Keys::validate_service_api_key( $key , $service  );
 	}
```

Comment:
I think we can omit this static method altogether since it's not directly used by this class anymore